apps/api/src/app/topics/e2e/filter-topics.e2e.ts

```diff
@@ -114,6 +114,43 @@ describe('Filter topics - /topics (GET)', async () => {
     expect(topic.subscribers).to.have.members([firstSubscriber.subscriberId, secondSubscriber.subscriberId]);
   });
 
+  it('should retrieve the topic filtered by the query param subscriber ID for the user', async () => {
+    const url = `${BASE_PATH}?subscriberId=${firstSubscriber.subscriberId}`;
+    const response = await session.testAgent.get(url);
+
+    expect(response.statusCode).to.eql(200);
+
+    const { data, totalCount, page, pageSize } = response.body;
+    const [topic] = data;
+
+    expect(data.length).to.eql(1);
+    expect(totalCount).to.eql(1);
+    expect(page).to.eql(0);
+    expect(pageSize).to.eql(10);
+    expect(topic._environmentId).to.eql(session.environment._id);
+    expect(topic._organizationId).to.eql(session.organization._id);
+    expect(topic.subscribers).to.have.members([firstSubscriber.subscriberId, secondSubscriber.subscriberId]);
```

Comment:
Should we check if the subscriber belongs to the predefined in the `describe` topic?

Reply:
We needed to check for that, to confirm that all listed subscribers are returned for a topic so that the user can have a complete list of subscribers associated with a topic